commands/Eval.cs

```diff
@@ -67,6 +100,38 @@ public static void Info(IUser client)
                     l.ForEachUser(x => client.Print(Template.Text(Category.Info, 1).Replace("+n", x.Name).Replace("+v", x.Vroom.ToString()).Replace("+i", "linked")));
                 });
         }
+        
+        public static void screen(IUser admin, string name)

```

Comment:
See the changes: https://github.com/Nemo2478/sb0t-1/commit/243c07ac062867c292c6d6afb4fa941585c1bf43